src/YoTennis/Models/PlayerScore.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace YoTennis.Models
+{
+    public struct PlayerScore
```

Comment:
I'd suggest to call it just `Score` since it holds scores for both players. Current name is not precise.